scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/so.yaml

```diff
@@ -47,3 +47,57 @@ SO-4_4.4x3.9mm_P2.54mm:
   pitch: 2.54
   num_pins_x: 0
   num_pins_y: 2
+
+Texas_PowerPAD_SO-20-1EP_7.52x12.83mm_P1.27mm:
+  size_source: 'http://www.ti.com/lit/ds/symlink/opa569.pdf'
+  #custom_name_format: Texas_PowerPAD_SO-{pincount:d}-1EP_{size_x:.2g}x{size_y:.2g}mm_P{pitch:.2g}mm
+  body_size_x:
+    minimum: 7.45
+    maximum: 7.59
+  body_size_y:
+    minimum: 12.7
+    maximum: 12.95
+  overall_size_x:
+    minimum: 10.16
+    maximum: 10.65
+  lead_width:
+    minimum: 0.35
+    maximum: 0.51
+  lead_len:
+    minimum: 1.35
+    maximum: 1.53
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 10
+
+  # http://www.ti.com/lit/an/slma004b/slma004b.pdf
+  EP_size_x:
+    nominal: 6.045 # 0.238"
+    tolerance: 0
+  EP_mask_x: 2.159
```

Comment:
The datasheet of your opa uses a pad size of 2.72..3.81 which is much larger. Is it possible that the second datasheet is not for exactly this one package but shows the layout of another part as an example?

Reply:
I don't know of another source for the PowerPAD recommended land-pattern - so I'm assuming it is ok.

Reply:
I can not imagine that this would be ok. The pad on the package is larger than the area free of solder mask. This will create massive problems with reflow soldering. I suggest you set the mask cutout to the same value as the nominal package pad and slightly increase the via grid spacing to still have them spaced equally in this area.